Scripts/reverse_integration_check.py

```diff
@@ -33,11 +34,14 @@ def main():
 
     logging.debug("ignoring tags before epoch_tag: %s" % epoch_tag)
 
-    tags_of_concern = [unmerged_tag for unmerged_tag in unmerged_tags if LooseVersion(unmerged_tag) > LooseVersion(epoch_tag)]
+    tags_of_concern = [tag for tag in unmerged_tags if LooseVersion(tag) > LooseVersion(epoch_tag)]
+
+    # Don't reverse integrate tags for adhoc builds
+    tags_of_concern = [tag for tag in tags_of_concern if "adhoc" not in tag]
 
     if len(tags_of_concern) > 0:
         logging.debug("Found unmerged tags newer than epoch: %s" % tags_of_concern)
-        raise RuntimeError("Found unmerged tags: %s" % tags_of_concern)
+        raise RuntimeError("💥 Found unmerged tags: %s" % tags_of_concern)
```

Comment:
I know it's annoying to start sticking emoji everywhere, but it's otherwise easy to miss this output since it gets jumbled with the existing git commit output.